source/install/install-tar.rst

```diff
@@ -1,8 +1,221 @@
-:nosearch:
-.. This page is intentionally not accessible via the LHS navigation pane because it's common content included on other docs pages.
+Install Mattermost Server using the tarball
+===========================================
 
-Install Mattermost server from a tarball
-----------------------------------------
+.. raw:: html
 
-.. include:: common-prod-deploy-tar.rst
-    :start-after: :nosearch:
\ No newline at end of file
+  <div class="mm-badge mm-badge--combo">
+
+    <div class="mm-badge__plan-deploy">
+      <p>
+        <img src="../_static/images/badges/flag_icon.svg" alt="" />
+        <span>Available on <a href="https://mattermost.com/pricing/">all plans</a></span>
+      </p>
+      <p>
+        <img src="../_static/images/badges/deployment_icon.svg" alt="" />
+        <span><a href="https://mattermost.com/download/">Self-hosted</a> deployments</span>
+      </p>
+    </div>
+
+    <div class="mm-badge__reqs">
+      <h3>Minimum system requirements:</h3>
+      <ul>
+        <li>Hardware: 1 vCPU/core with 2GB RAM (support for up to 1,000 users)</li>
+        <li>Database: PostgreSQL v11+</li>
+        <li>Network:
+          <ul>
+            <li>Application 80/443, TLS, TCP Inbound</li>
+            <li>Administrator Console 8065, TLS, TCP Inbound</li>
+            <li>SMTP port 10025, TCP/UDP Outbound</li>
+          </ul>
+        </li>
+      </ul>
+    </div>
+
+  </div>
+
+You can install the Mattermost Server on any 64-bit Linux system using the tarball. Deployment includes 4 steps: `download <#download-the-latest-mattermost-server-tarball>`__, `install <#install>`__, `setup <#setup>`__, and `updates <#updates>`__.
+
+.. important::
+
+    - This is the most flexible installation method, but it comes with the highest effort, normally favored by advanced system adminsitrators. 
+    - If you are running the Mattermost Server and database a single system, we recommend the `Mattermost Omnibus install method </install/installing-mattermost-omnibus.html>`__ as this greatly reduces setup and ongoing maintenance, and using the Mattermost PPA for a scalable installation. More modern installation methods such as the Mattermost Helm Chart or Kubernetes Operator are available and are highly recommended.
+
+
+Download the latest Mattermost Server tarball
+---------------------------------------------
+
+1. In a terminal window, ssh onto the system that will host the Mattermost Server. 
+
+2. Using ``wget``, download the Mattermost Server release you want to install.
+
+Download the latest release
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: none
+
+    wget https://releases.mattermost.com/7.10.0/mattermost-7.10.0-linux-amd64.tar.gz
+
+Download the current Extended Support Release (ESR)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: none
+    
+    wget https://releases.mattermost.com/7.8.3/mattermost-7.8.3-linux-amd64.tar.gz
```

Comment:
@emdecr this is the copy action that needs the button (can be the same as the first)